Database/Corrections/sodQuestFixes.lua

```diff
@@ -1434,6 +1434,21 @@ function SeasonOfDiscovery:LoadQuests()
         },
         -- End Duskwood Incursion
 
+        [81764] = { -- The Mysterious Merchant
+            [questKeys.preQuestSingle] = {79970},
+            [questKeys.nextQuestInChain] = 81765,
+            [questKeys.zoneOrSort] = zoneIDs.DUSTWALLOW_MARSH,
+        },
+        [81765] = { -- Elrick, Paladin of the Silver Hand
+            [questKeys.preQuestSingle] = {81764},
+            [questKeys.nextQuestInChain] = 81766,
+            [questKeys.zoneOrSort] = zoneIDs.DUSTWALLOW_MARSH,
+        },
+        [81766] = { -- The Bloody Missive
+            [questKeys.preQuestSingle] = {81765},
```

Comment:
```suggestion
        [81766] = { -- The Bloody Missive
            [questKeys.startedBy] = {{6272},nil,{219930}},
            [questKeys.preQuestSingle] = {81765},
```

We add both, the item that starts the quest and the NPC which drops the item, because currently Questie does not show quests, which only have an item starter.